[2025-03-16T13:41:38.727+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-16T13:41:38.798+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: read_csv_file_dag.run_script_task manual__2025-03-16T13:41:36.498458+00:00 [queued]>
[2025-03-16T13:41:38.828+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: read_csv_file_dag.run_script_task manual__2025-03-16T13:41:36.498458+00:00 [queued]>
[2025-03-16T13:41:38.831+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-03-16T13:41:38.888+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): run_script_task> on 2025-03-16 13:41:36.498458+00:00
[2025-03-16T13:41:38.904+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=275) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-16T13:41:38.912+0000] {standard_task_runner.py:72} INFO - Started process 276 to run task
[2025-03-16T13:41:38.914+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'read_csv_file_dag', 'run_script_task', 'manual__2025-03-16T13:41:36.498458+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpblallhom']
[2025-03-16T13:41:38.919+0000] {standard_task_runner.py:105} INFO - Job 24: Subtask run_script_task
[2025-03-16T13:41:39.105+0000] {task_command.py:467} INFO - Running <TaskInstance: read_csv_file_dag.run_script_task manual__2025-03-16T13:41:36.498458+00:00 [running]> on host e8ff8b4f9a7d
[2025-03-16T13:41:39.429+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='read_csv_file_dag' AIRFLOW_CTX_TASK_ID='run_script_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T13:41:36.498458+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T13:41:36.498458+00:00'
[2025-03-16T13:41:39.432+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-16T13:41:39.611+0000] {logging_mixin.py:190} INFO - Script Output: 
[2025-03-16T13:41:39.613+0000] {python.py:240} INFO - Done. Returned value was: 
[2025-03-16T13:41:39.693+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-16T13:41:39.696+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=read_csv_file_dag, task_id=run_script_task, run_id=manual__2025-03-16T13:41:36.498458+00:00, execution_date=20250316T134136, start_date=20250316T134138, end_date=20250316T134139
[2025-03-16T13:41:39.784+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-16T13:41:39.857+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-16T13:41:39.863+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
